src/components/WinWallForm/WinWallForm.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState, useEffect } from "react";
 import { Link, useParams } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 import "./WinWallForm.css";
 
```

Comment:
try adding variables  amd all win-wall fields to your const, eg:
  const [project, setProject] = useState({
    title: "",
    blurb: "",
    description: "",
    category_id: "",
    goal: "",
    goal_date: "",
    primary_image: "",
    secondary_image: "",
    is_open: "",
    is_archived: "",
    pledge_type_id: "",
  });

note- should be in line with your 'prevWinwall'  in handle submit, eg: Winwall and setWinwall